conference/admin.py

```diff
@@ -991,26 +1009,31 @@ def _assigned(self, ticket):
     _assigned.allow_tags = True
     _assigned.admin_order_field = 'p3_conference__assigned_to'
 
-    def changelist_view(self, request, extra_context=None):
-        if not request.GET:
-            q = request.GET.copy()
-            q['fare__conference'] = settings.CONFERENCE_CONFERENCE
-            q['fare__ticket_type__exact'] = 'conference'
-            request.GET = q
-            request.META['QUERY_STRING'] = request.GET.urlencode()
-        return super(TicketAdmin,self).changelist_view(request, extra_context=extra_context)
-
-    def get_queryset(self, request):
-        qs = super(TicketAdmin, self).get_queryset(request)
-        qs = qs.select_related('user', 'fare',)
-        return qs
-
-    def get_urls(self):
-        urls = super(TicketAdmin, self).get_urls()
-        my_urls = [
-            re_path(r'^stats/data/$', self.admin_site.admin_view(self.stats_data_view), name='conference-ticket-stats-data'),
-        ]
-        return my_urls + urls
+    # XXX: These are redefined further below (I commented the older ones for now)
+    # def changelist_view(self, request, extra_context=None):
+    #     if not request.GET:
+    #         q = request.GET.copy()
+    #         q['fare__conference'] = settings.CONFERENCE_CONFERENCE
+    #         q['fare__ticket_type__exact'] = 'conference'
+    #         request.GET = q
+    #         request.META['QUERY_STRING'] = request.GET.urlencode()
+    #     return super(TicketAdmin, self).changelist_view(request, extra_context=extra_context)
+
+    # def get_queryset(self, request):
+    #     qs = super(TicketAdmin, self).get_queryset(request)
+    #     qs = qs.select_related('user', 'fare',)
+    #     return qs
+
+    # def get_urls(self):
+    #     urls = super(TicketAdmin, self).get_urls()
+    #     my_urls = [
+    #         re_path(
+    #             r'^stats/data/$',
+    #             self.admin_site.admin_view(self.stats_data_view),
+    #             name='conference-ticket-stats-data'
+    #         ),
+    #     ]
+    #     return my_urls + urls
```

Comment:
@umgelurgel you redefined these methods further down in the code. Can you take a look and check what's this about?

Reply:
Hey, thanks for catching this - the admin configuration was spread across modules, I've moved it to one module in an earlier MR - mussed have missed the duplicates here. I'ts safe to remove the commented out parts.